matisse/src/main/java/com/zhihu/matisse/internal/ui/widget/CheckView.java

```diff
@@ -188,7 +192,11 @@ private void initBackgroundPaint() {
             mBackgroundPaint = new Paint();
             mBackgroundPaint.setAntiAlias(true);
             mBackgroundPaint.setStyle(Paint.Style.FILL);
-            mBackgroundPaint.setColor(Color.parseColor("#1E8AE8"));
+            TypedArray ta = getContext().getTheme()
+                    .obtainStyledAttributes(new int[]{R.attr.item_checkCircle_backgroundColor});
+            int color = ta.getColor(0, 0);
```

Comment:
Nope, it's okay. I'll have it ready as soon as possible. Do I have to create a new pull request after I've pushed the changes to my fork?